src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -17858,5 +17859,29 @@ public void PrintCoordinates()
                 Diagnostic(ErrorCode.ERR_PredefinedTypeMemberNotFoundInAssembly, "(int x2, int y2) = GetCoordinates2();").WithArguments("Item2", "System.ValueTuple<T1, T2>", "comp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null").WithLocation(26, 9)
             );
         }
+
+        [Fact, WorkItem(13494, "https://github.com/dotnet/roslyn/issues/13494")]
+        public static void Bug13494()
+        {
+            var source1 = @"
+class Program
+{
+    static void Main(string[] args)
+    {
+        var(a,  )
+    }
+}
+";
+
+            var text = SourceText.From(source1);
+            var startTree = SyntaxFactory.ParseSyntaxTree(text);
+            var finalString = startTree.GetCompilationUnitRoot().ToFullString();
+
+            var newText = text.WithChanges(new TextChange(new TextSpan(80, 0), " ")); // add space before closing-paren
```

Comment:
>var [](start = 12, length = 3)

This test will probably not work on some platforms (because line endings change between LF and CRLF). I recommend replacing 80 with the result of a string search.